### DISCUSSION TITLE
Tablet not letting me access full screen

### DISCUSSION THREAD

**Discussion by TheSeaCat** [ppy/osu#21974] [Q&A]
When I use my stylus in gameplay or just the menu in osu lazer, I can't access the full screen. When i bring my stylus to the top area of my tablet area, it only lets me access the middle of the screen. I have the osu lazer tablet driver turned off, and this didn't happen until I did. 
Video of problem: https://streamable.com/1rwxvt

**Comment by bdach** [comment]
- What OS is this?
- What is the tablet?
- Do you have multiple screens?

**Comment by TheSeaCat** [comment]
I am on macOS , the tablet is a Wacom intuos M, and I use two screens.

**Reply by bdach** [reply]
How are the screens positioned in relation to each other? Remember that the tablet working area will span all active screens, which may lead to this

**Reply by TheSeaCat** [reply]
I usually play osu with the one on the right, but the problem is usually just vertical

**Comment by Inconsist** [comment]
You can try to disable the high percision mosue it fixed it for me on windows.
